crates/cext/cbindgen.toml

```
@@ -79,3 +79,4 @@ prefix = "Qk"
 "CDelayUnit" = "DelayUnit"
 "CNeighbors" = "Neighbors"
 "COperationKind" = "OperationKind"
+CDagNeighbors = "DagNeighbors"
```

```suggestion
"CDagNeighbors" = "DagNeighbors"
```

technically completely fine as it was before, the suggestion is just for consistency with the others